requirements.txt

```diff
@@ -21,6 +21,7 @@ scipy
 sphinx
 sphinxcontrib-apidoc
 recommonmark
+gdown
```

Comment:
I think this change also needs to be made in `setup.py` ?

Reply:
oh yes, updated